src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -205,7 +205,7 @@ public function __construct(BufferingLogger $bootstrappingLogger = null, bool $d
      * @param array|int       $levels  An array map of E_* to LogLevel::* or an integer bit field of E_* constants
      * @param bool            $replace Whether to replace or not any existing logger
      */
-    public function setDefaultLogger(LoggerInterface $logger, $levels = \E_ALL, bool $replace = false): void
+    public function setDefaultLogger(LoggerInterface $logger, array|int $levels = \E_ALL, bool $replace = false): void
```

Comment:
Looking at line 219, this can also be `null`
```suggestion
    public function setDefaultLogger(LoggerInterface $logger, array|int|null $levels = \E_ALL, bool $replace = false): void
```